modules/tribes/client/components/TribesList.component.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import Tribe from './Tribe';
+import SuggestTribe from './SuggestTribe';
+
+const List = styled.ul`
+  margin: -5px -5px 15px -5px;
+  line-height: 0;
+`;
+
+const Item = styled.li`
+  line-height: 1.42857143; // set back the line-height set by bootstrap
```

Comment:
Similar concerns with this value than in https://github.com/Trustroots/trustroots/pull/1144/files#r362089900

Does the comment mean that Bootstrap sets different line-height for `li` than rest of elements?

Reply:
The line height had been set to this value by bootstrap before the migration. Here we took that responsibility away from bootstrap, so we entered the value manually to keep the original looks.
https://stackoverflow.com/questions/19982651/why-does-bootstrap-set-the-line-height-property-to-1-428571429
I'll try to improve the comment. Or do you have ideas how else to improve this, @simison?

Reply:
Oh, hm, I'd really prefer we inherit those kind of styles from UI library (currently bootstrap) instead of manually entering them everywhere. :-/

Can we use Bootstrap classnames and inherit right base styles?

Reply:
I don't understand why i set `line-height: 0` on `ul` in the first place. Removing that removes the necessity of specifying this weird value here.

I don't like writing this either. It was a lazy fix.

Reply:
Fixed in 9fcdd0ab1e89d508057425b433571a1394c42565.